05-onchain.md

```diff
@@ -56,8 +56,8 @@ the output is considered to be its own *resolving* transaction.
 
 Outputs that are *resolved* are considered *irrevocably resolved*
 once the remote's *resolving* transaction is included in a block at least 100
-deep, on the most-work blockchain. 100 blocks is far greater than the
-longest known Bitcoin fork and is the same wait time used for
+deep, on the most-work blockchain. 100 blocks are far greater than the
```

Comment:
i actually think `is` is clearer here, as `100 blocks` is a singular entity. (but i'm admittedly bending grammar rules a smidge here)

Reply:
how about `the length of 100 blocks is far greater than...`?
I read it several times.

> 100 blocks is far greater than the longest known Bitcoin fork

Now I start to think `100 blocks is greater than Bitcoin fork` doesn't make sense. (block greater than fork???)